web/models.py

```
@@ -2660,6 +2660,138 @@ def __str__(self):
+
+class WorkSubmission(models.Model):
+    """Model for storing work submissions for review."""
+
+    STATUS_CHOICES = [
+        ("submitted", "Submitted"),
+        ("under_review", "Under Review"),
+        ("reviewed", "Reviewed"),
+        ("needs_revision", "Needs Revision"),
+        ("approved", "Approved"),
+        ("rejected", "Rejected"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)